### PR TITLE
W-12397480-Kerberos-support

### PR DESCRIPTION
**Pull request by luanamulesoft**
# Writer's Quality Checklist

Before merging your PR, did you:

- [x] Run spell checker
- [x] Run link checker to check for broken xrefs
- [x] Check for orphan files
- [x] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: runtime-manager/modules/ROOT/pages/rtm-agent-proxy-config.adoc [hunk 1]
```diff
@@ -35,13 +35,24 @@ To configure the proxy server connection:
...
++
+For Kereros authentication, use the new parameter to specify that this protocol will be used along with the configuration file paths.
++
+[source,console,linenums]
+----
+$MULE_HOME/bin/amc_setup -H token server-name -P proxy-host proxy-port proxy-user proxy-password --use-kerberos-proxy krb5-configuration-path jaas-configuration-path
+----
 
-If the proxy server doesn't require authentication, omit _proxy-user_ and _proxy-password_.
+If the proxy server doesn't require authentication, omit _proxy-user_ and _proxy-password_. These options are required when using Kerberos authentication via _--use-kerberos-proxy_.
```
Comment (lggomezsf) [inline runtime-manager/modules/ROOT/pages/rtm-agent-proxy-config.adoc:52]: @rodrisouto this last line isn't true any longer right?
Otherwise the PR LGTM

**Review by rodrisouto — DISMISSED** [review]
lgtm

**Review by lggomezsf — CHANGES_REQUESTED** [review]

**Review by lggomezsf — DISMISSED** [review]

File: runtime-manager/modules/ROOT/pages/rtm-agent-proxy-config.adoc (hunk 1 above)
Comment (rodrisouto) [reply runtime-manager/modules/ROOT/pages/rtm-agent-proxy-config.adoc:52]: why is it not true? the user and password are required for `--use-kerberos-proxy`

File: runtime-manager/modules/ROOT/pages/rtm-agent-proxy-config.adoc [hunk 2]
```diff
@@ -13,7 +13,7 @@ include::reuse::partial$non-inclusive-banner.adoc[]
 
 If your Mule runtime engine runs inside a firewall that restricts external communication through a proxy server, you can configure the Runtime Manager agent to route traffic through the proxy server to Runtime Manager.
 
-The Runtime Manager agent supports only the basic access authentication method.
+The Runtime Manager agent supports supports the basic access and Kerberos v5 (via SPNEGO) authentication schemes for proxied communication.
```
Comment (sathya0) [inline runtime-manager/modules/ROOT/pages/rtm-agent-proxy-config.adoc]:
```suggestion
The Runtime Manager agent supports the basic access and Kerberos v5 (via SPNEGO) authentication schemes for proxied communication.
```

File: runtime-manager/modules/ROOT/pages/rtm-agent-proxy-config.adoc [hunk 3]
```diff
@@ -35,13 +35,24 @@ To configure the proxy server connection:
...
 . Run this command:
 +
+For basic authentication, the -P parameter sets up the authenticated proxy.
++
 [source,console,linenums]
 ----
 $MULE_HOME/bin/amc_setup -H token server-name -P proxy-host proxy-port proxy-user proxy-password
 ----
++
+For Kereros authentication, use the new parameter to specify that this protocol will be used along with the configuration file paths.
```
Comment (sathya0) [inline runtime-manager/modules/ROOT/pages/rtm-agent-proxy-config.adoc]:
```suggestion
For Kerberos authentication, use the new parameter to specify that this protocol will be used along with the configuration file paths.
```

**Review by sathya0 — APPROVED** [review]

**Review by lggomezsf — APPROVED** [review]

**Pull request merged**